source/material_model/grain_size.cc

```diff
@@ -211,6 +211,55 @@ namespace aspect
 
 
 
+    namespace
+    {
+      // This function returns the log normal distribution of the grain size.
+      // Description can be found in eq 8 of Bercovici and Ricard (2012)
+      double
+      moment_of_grain_size_distribution (const unsigned int n)
+      {
+        const double sigma = 0.8;
```

Comment:
Can you add a comment saying that this is the variance of the log-normal distribution?